runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -57,6 +57,7 @@
  */
 class XWalkContent implements XWalkPreferencesInternal.KeyValueChangeListener {
     private static String TAG = "XWalkContent";
+    private static final String namespace = "http://schemas.android.com/apk/res-auto";
```

Comment:
`namespace` should be capitalized like others, and we better use a more explicit name.